docs/testcases/UI/Cluster_nodes/Hot_node_pools/2414/2414_2.md

```diff
@@ -0,0 +1,29 @@
+# [Manual] Notify administrators if several node pools reached its max size
+
+Test verifies that the notification shall be single for all pools if there are several of them reach its max size.
+
+**Prerequisites**:
+- Admin user
+- Non-admin user
+- Perform [2414_1](2414_1.md) case
+
+**Preparations**:
+1. Repeat steps 1-9 of the Preparations of [2414_1](2414_1.md) case 2 times with different docker images (*tool1* and *tool2*)
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Select test *tool1* | |
+| 2 | Repeat steps 7-15 of the [2414_1](2414_1.md) case | |
+| 3 | Repeat steps 1-2 | |
+| 4 | Select test *tool2* | |
+| 5 | Repeat step 2 | | 
```

Comment:
For more understandable, fix to `Repeat steps 7-15 of the [2414_1](2414_1.md) case`

Reply:
Done